lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1249,7 +1267,11 @@ def _resource_sku_to_capability(
 
         # all nodes support following features
         node_space.features.update(
-            [features.StartStop.name(), features.SerialConsole.name()]
+            [
+                features.StartStop.name(),
+                features.SerialConsole.name(),
+                features.DiskStandardLRS.name(),
```

Comment:
@squirrelsc The users would need to delete `azure_location_<location>.cache` otherwise their Lisa run would fail because of this change. Is there a way to make this process more streamlined for users?

Reply:
The cache will expire every 7 days. May be we can change the cache to one day? We may add version into the file in future. But for now, it's acceptable. If someone fails on this, we can let them know. At most 7 days, after the cache refreshed, it's mitigated.

Reply:
Changing cache to 1 day is a good idea for minimum impact on users. I will add a separate commit to update this value.